app/test_split_integer.py

```diff
@@ -1,21 +1,129 @@
+import pytest
+
+
 from app.split_integer import split_integer
 
 
-def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+@pytest.mark.parametrize(
+    "value,number_of_parts",
+    [
+        (1, 1),
+        (6, 2),
+        (17, 4),
+        (32, 5),
+        (100, 20)
+    ]
+)
+def test_max_and_min_difference_should_be_less_that_one(
+    value: int,
+    number_of_parts: int
+) -> None:
+    result = split_integer(value, number_of_parts)
+    assert max(result) - min(result) <= 1, (
+        "Difference of max and min number in the array should be <= 1"
+    )
+
```

Comment:
Please read checklist before pushing your solution!
Don't create unnecessary variables for simple tests:

Reply:
FIx it in all places